docs/README.md

```diff
@@ -99,3 +99,55 @@ $ git commit
 Create a pull request once you are ready to submit your changes.
 
 We will review your changes, add any feedback and once everything looks good merge your changes into the mainline.
+
+
+## Advanced Usage
+
+### Re-generating diagrams for API docs
+
+Generate new grammars and diagrams when updating the docs as follows:
+
+_Note: Modifications will typically be in the `docs/content/latest/api/` either `ysql` or `ycql` subfolder.
+Will assume that the subfolder is `docs/content/latest/api/ysql` and the target file is `commands/cmd_copy.md` in the examples below_
```

Comment:
In the examples below, the subdirectory is `docs/content/latest/api/ysql` and the target file is `commands/cmd_copy.md`._